packages/shared/src/components/cards/PostList.tsx

```diff
@@ -15,6 +15,7 @@ import PostAuthor from './PostAuthor';
 import FeedItemContainer from './FeedItemContainer';
```

Comment:
This looks a bit different then the design for article cards.
Article cards:
<img width="1297" alt="Screenshot 2023-12-05 at 4 02 49 PM" src="https://github.com/dailydotdev/apps/assets/36197304/252258d3-884e-4823-916d-822cc8f3fe4b">

New design:
<img width="903" alt="Screenshot 2023-12-05 at 4 08 41 PM" src="https://github.com/dailydotdev/apps/assets/36197304/03de4ac1-ab47-45f5-b059-9dd812bf50e7">

Not sure if it is intended to be changed here as it's not mentioned in the DR either, will check with design.

Reply:
Confirmed with design & @rebelchris  - this will stay with the old version and the change will be tackled globally for all cards in a different ticket

Reply:
Should be able to merge it now to new design :) (Actually should just work out of the box now)